modules/nf-core/sentieon/tnscope/tests/nextflow.config

```diff
@@ -0,0 +1,28 @@
+process {
+
+    withName: "TUMOUR_ADDORREPLACEREADGROUPS" {
+        ext.args = "--CREATE_INDEX true -ID tumour -LB tumour -PL ILLUMINA -PU bc1 -SM tumour"
```

Comment:
Should this be set with `params.tumor_addorreplacereadgroups_args` ?

Reply:
I don't think it is necessary to introduce a parameter when it is only being used once personally.

Reply:
The point of the modules_args IMO was so that you didn't have to have seperate config files for different tests, not to force everything to use it.

Reply:
Okay make sense !